.github/workflows/uno-updater.yml

```
@@ -20,12 +19,29 @@ on:
 jobs:
   manifest-update:
     runs-on: windows-latest
+    strategy:
+      matrix:
+        branch:
+          - main
+          - release/stable/5.3
```

Let's use that instead, I thought it was used by triggers, but it's not.

```suggestion
          - release/stable/*
```

---

Scratch that, wrong line.